rust_snuba/src/mutations/clickhouse.rs

```diff
@@ -170,6 +170,76 @@ fn format_query(table: &str, batch: &MutationBatch) -> Vec<Vec<u8>> {
     ]
 }
 
+struct ClickhouseTestClient {
+    url: String,
+    table: String,
+    client: Client,
+}
+
+impl ClickhouseTestClient {
+    pub async fn new(table: String) -> anyhow::Result<Self> {
+        // hardcoding local Clickhouse settings
```

Comment:
i think we want to make those environment variables with these defaults. however, i think the 9000 port is also wrong, you want 8123

Reply:
oh yeah i was testing this out and forgot to revert. 8123 is correct